v19.2/table-expressions.md

```diff
@@ -87,6 +87,8 @@ For example:
 
 {% include {{page.version.version}}/misc/force-index-selection.md %}
 
+You can also force index selection for [`DELETE`](delete.html#delete-on-a-specific-index) and [`UPDATE`](update.html#update-on-a-specific-index) statements.
+
```

Comment:
I see what you did there since the preceding snippet is also used in the SELECT docs.  Maybe this could be an info callout so it's harder to miss on this page?